mlflow/store/artifact/unity_catalog_oss_models_artifact_repo.py

```diff
@@ -79,7 +81,8 @@ def __init__(self, artifact_uri, registry_uri):
         )
         if registry_uri_from_artifact_uri is not None:
             registry_uri = registry_uri_from_artifact_uri
-        _, key_prefix = get_db_info_from_uri(registry_uri)
+
+        _, key_prefix = get_db_info_from_uri(urllib.parse.urlparse(artifact_uri).path)
```

Comment:
Wait.  Is this correct?  Isn't the registry uri the one that would be `uc:databricks-uc:<something>`?

Reply:
Whoops, I meant to use the registry_uri here. Should be fixed now.